src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = 40 * days;
+  } else if (days >= 3 && days < 7) {
+    total = (40 * days) - 20;
+  } else if (days <= 7) {
+    total = (40 * days) - 50;
+  }
```

Comment:
🚨 Your condition is wrong - `days <= 7` is need to be `days >= 7`.
Also this part can be simplified to only 2 options: first check if `days >= 7`, after it if `days >= 3`. Remember, you can immediately return result from inside `if (days >= 7) {return}` and stop the execution.